specs/binding-kafka.spec/src/main/resources/META-INF/zilla/kafka.idl

```
@@ -219,6 +226,7 @@ scope kafka
         {
             int32 deferred = 0;         // INIT only (TODO: move to DATA frame)
             int64 timestamp = 0;        // INIT only
+            int64 filters = 0;          // INIT only
```

Suggest we default to `-1` instead, as filters mask `0` means no match.